stdlib/java/src/main/ballerina/src/java/Module.md

```diff
@@ -1,4 +1,3 @@
-## Module overview
-
 This module provides the API for Java interoperability in Ballerina. It includes a set of Ballerina annotations with which Java constructors, methods and fields can provide implementations to Ballerina functions with external function bodies.  
 
+For information on the operations, which you can perform with the java module, see the below **Functions**. For examples on the usage of the operations, see [Create Java Objects](https://ballerina.io/v1-2/learn/by-example/create-java-objects.html), [Invoke Java Methods](https://ballerina.io/v1-2/learn/by-example/invoke-java-methods.html), [Access/Mutate Java Fields](https://ballerina.io/v1-2/learn/by-example/access-mutate-java-fields.html), [Java Varargs](https://ballerina.io/v1-2/learn/by-example/java-varargs.html), [Overloaded Methods/Constructors](https://ballerina.io/v1-2/learn/by-example/overloaded-methods-constructors.html), [Java Exceptions](https://ballerina.io/v1-2/learn/by-example/java-exceptions.html)
```

Comment:
```suggestion
For information on the operations, which you can perform with the java module, see the below **Functions**. For examples on the usage of the operations, see the [Create Java Objects Example](https://ballerina.io/learn/by-example/create-java-objects.html), [Invoke Java Methods Example](https://ballerina.io/learn/by-example/invoke-java-methods.html), [Access/Mutate Java Fields Example](https://ballerina.io/learn/by-example/access-mutate-java-fields.html), [Java Varargs Example](https://ballerina.io/learn/by-example/java-varargs.html), [Overloaded Methods/Constructors Example](https://ballerina.io/learn/by-example/overloaded-methods-constructors.html), [Java Exceptions Example](https://ballerina.io/learn/by-example/java-exceptions.html)
```
Shall we add the BBE list in bullets for more clarity?

Reply:
I don't think bullets are needed and it breaks the consistency. Having such links are common so the user won't get confused
![Selection_124](https://user-images.githubusercontent.com/8381003/79036179-56117a00-7be3-11ea-8afc-cf03bd6d1173.png)

Reply:
The separation is not very clear. i think we need bullets since this list is big.